astropy/stats/funcs.py

```diff
@@ -835,7 +885,8 @@ def mad_std(data, axis=None, func=None):
     """
 
     # NOTE: 1. / scipy.stats.norm.ppf(0.75) = 1.482602218505602
```

Comment:
This is unrelated to this PR, but it would be good to mention this factor in the docstring; if I understand correctly, it essentially is the exact number for the following:
```
1./np.median(np.abs(np.random.normal(size=100000)))
# 1.4832115241968997
```